e2e/src/step-definitions/media-viewer.ts

```diff
@@ -1,24 +1,53 @@
-import { Given, Then } from 'cucumber';
-import { AppPage } from '../pages/app.po';
-import { by } from 'protractor';
+import {When, Given, Then} from 'cucumber';
+import {AppPage} from '../pages/app.po';
+import {by} from 'protractor';
 import * as chai from 'chai';
+import {NavigatePage} from '../pages/navigate.po';
 
 const page = new AppPage();
+const navigatePage: NavigatePage = new NavigatePage();
 
-Given('I am on Media Viewer Page',  async () => {
+Given('I am on Media Viewer Page', async () => {
   await page.preparePage();
 });
 
 Given('I enable toggle buttons', async () => {
```

Comment:
this seems like an action, so shouldn't it be When(....)

Reply:
fixed